README.md

```diff
@@ -1,9 +1,17 @@
 # whitehall
 
+### Running the Application
+
+**Use GOV.UK Docker to run any commands that follow.**
+
 Whitehall is deployed in two modes:
 
-- 'admin' for publishers to create and manage content; and
-- 'frontend' for rendering some content under https://www.gov.uk/government and https://www.gov.uk/world.
+- 'admin' for publishers to create and manage content (e.g. <http://whitehall-admin.dev.gov.uk/government/admin/news/new>) and
+- 'frontend' for rendering some content under https://www.gov.uk/government, https://www.gov.uk/world & http://www.gov.uk/government/get-involved
+
+Traditionally, the two sides of Whitehall are available on different domains in development, which reflect their counterparts in production:
+
+While this usually results in different routing behaviour, in development [all routes can be accessed from either domain](https://github.com/alphagov/whitehall/blob/530abc13018145a6efe6ab4a19f6210254e2e304/config/routes.rb#L3-L5), although [the redirect behaviour may differ](https://github.com/alphagov/whitehall/blob/530abc13018145a6efe6ab4a19f6210254e2e304/config/routes.rb#L25-L28).
```

Comment:
These links tie these lines to a specific git sha which cretes risk of those files change downstream. Could we change this to reference `main` instead? Eg: https://github.com/alphagov/whitehall/blob/main/config/routes.rb#L3-L5

Reply:
Makes sense, added